charts/stable/qbitrr/docs/installation-notes.md

```diff
@@ -32,11 +32,11 @@ If `CompletedDownloadFolder` option is not set to `"/completed_downloads"` qBitr
 
 ### qBit
 
-This section has qBittorrent specific settings. Ensure that `Host`, `Port`, `UserName`, and `Password` all match your qBittorrent installation. The host and port can be located using the Heavyscript `heavyscript dns -a` command.
+This section has qBittorrent specific settings. Ensure that `Host`, `Port`, `UserName`, and `Password` all match your qBittorrent installation. The host and port can be located using the Heavyscript `heavyscript dns -a` command. It is not recommended to use internal links for this part.
```

Comment:
Why it is not recommended?

Reply:
qBittorrent internal linking works fine.

Reply:
Good point, reading the installation notes I got that impression since I didn't understand how internal linking worke. My bad, I'll fix it.